README.rst

```diff
@@ -45,6 +45,11 @@ Purchasable Content Types
 Types can be Dexterity or Archetypes. Make your own content type purchasable using a behavior or schema extender.
 (Note: Archetypes support in Python 2.x only)
 
+Related products
```

Comment:
[@silviot] I think the README should say "Recommended Products" instead of "Related products". And the text should use the "recommended" idea too since we are specifically going for an Amazon-like cart recommendations feature, not a generic related items feature. Maybe:

A product can have other products related to it so that they can be recommended at the time of purchase. When users review their cart, they will be presented with a list of recommended products populated from the Recommended Products fields of all the products they're about to purchase.

Also I wonder if we should have a maximum number of recommendations that are shown? If someone is purchasing 5 things and they each have 3 recommended items that might be too long a list to show.

But probably better to put a basic feature out there and then get enhancement requests.

Also I would suggest that to avoid confusion you change all the `related_products` and similar names to `recommended_products`. Although I'm open to arguments against that.